src/mmw/apps/modeling/migrations/0004_merge.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not sure why 003/4 migrations were generated - they look like no-ops.  Can they be removed?